xml/http_servers.xml

```diff
@@ -1506,6 +1506,19 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:nginx:nginx:{service.version}"/>
   </fingerprint>
 
+  <fingerprint pattern="^WebSocket\+\+/([\d.-]+(?:-dev)?)$">
+    <description>WebSocket++ web server - https://github.com/zaphoyd/websocketpp</description>
+    <example service.version="0.8.1">WebSocket++/0.8.1</example>
+    <example service.version="0.8.2">WebSocket++/0.8.2</example>
+    <example service.version="0.4.0">WebSocket++/0.4.0</example>
+    <example service.version="0.8.0-dev">WebSocket++/0.8.0-dev</example>
+    <example service.version="2011-09-25">WebSocket++/2011-09-25</example>
```

Comment:
The date value doesn't appear to be valid version of this software. See zaphoyd/websocketpp
[changelog.md](https://github.com/zaphoyd/websocketpp/blob/master/changelog.md)
> 0.3.0-alpha1 - 2013-06-09
> - Initial Release

Reply:
It's a live server header via censys

Reply:
https://search.censys.io/search?resource=hosts&sort=RELEVANCE&per_page=25&virtual_hosts=EXCLUDE&q=same_service%28not+services.software.product%3A*+and+services.service_name%3A+HTTP+and+services.http.response.headers.server%3AWebSocket%2B%2B%2F2011-09-25%29

Reply:
Do we have any additional evidence it is actually WebSocket++?

Reply:
Although all of them are in China, which often has questionable server values (such as "iis8.0"). It's possible they're honeypots or simply faked for whatever reason.

Reply:
I took a deep dive into the commit history of zaphoyd/websocketpp and found `WebSocket++/2011-09-25` hardcoded! 
https://github.com/zaphoyd/websocketpp/blob/0568bbe1126478fadeeb52f7fafe6ed49608b23f/src/websocket_server_session.cpp#L285